server/Main.hs

```diff
@@ -72,7 +73,8 @@ server externs initEnv port = do
             Right m | P.getModuleName m == P.ModuleName [P.ProperName "Main"] -> do
               (resultMay, ws) <- runLogger' . runExceptT . flip runReaderT P.defaultOptions $ do
                 ((P.Module ss coms moduleName elaborated exps, env), nextVar) <- P.runSupplyT 0 $ do
-                  desugared <- P.desugar externs [P.importPrim m] >>= \case
+                  env <- fmap fst . runWriterT $ foldM P.externsEnv P.primEnv externs
```

Comment:
This builds the externs for every compilation, right? Is it possible to build this once and reuse it?

Reply:
Ah of course, yes it does.